src/main/webapp/nearyou.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Social Map</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBf5E9ymEYBv6mAi78mFBOn8oUVvO8sph4&libraries=places"></script>
+    <script src="profileScript.js"></script>
+    <script src="eventScript.js"></script>
+    <script src="api_key.js"></script>
+    <script src="mapScript.js"></script>
+    <script src="nearmeScript.js"></script>
```

Comment:
You may also want to rename `nearmeScript.js` and other instances of `nearme` in this file.